src/appleseed/renderer/modeling/bsdf/microfacethelper.h

```diff
@@ -275,6 +295,14 @@ class MicrofacetBRDFHelper
 
         return false;
     }
+
+    static void flip_directions(
```

Comment:
Looking at what this method does, I feel the name is not entirely adequate.

Reply:
I removed it an inlined the code manually.